server/routes/place/html.py

```diff
@@ -66,6 +67,49 @@
     'Continent',
 }
 
+# Location of manually written templates for SEO experimentation in GCS bucket
+SEO_EXPERIMENT_HTML_GCS_DIR = "seo_experiments/active"
+
+# Location of manually written templates for SEO experimentation on local disk
+SEO_EXPERIMENT_HTML_LOCAL_DIR = "config/seo_experiments/html_templates/active/"
+
+# Map DCID of SEO experiment places to their template filename
+SEO_EXPERIMENT_DCID_TO_HTML = {"country/EGY": "Egypt.html"}
+
+
+def get_seo_experiment_template(dcid: str) -> str:
+  """Load page template for SEO experiments
+  
+  Will load files from GCS unless server is running locally, in which case
+  will load from server/config/seo_experiments/html_templates/active instead.
+
+  If a file cannot be found or read, will return empty string instead.
+  """
+  experiment_template = ""
+  try:
+    filename = SEO_EXPERIMENT_DCID_TO_HTML[dcid]
+
+    if current_app.config['LOCAL']:
```

Comment:
How would we test loading templates through GCS when running locally?

Reply:
Switched to defaulting to reading from GCS so we can also test the loading templates path, and instead added an optional `useLocalTemplates=true` param to the request to enable reading files locally when I'm developing new templates.